src/pages/feedback.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { makeStyles, Typography } from "@material-ui/core";
+import BackButton from "../comps/ui/BackButton";
+import SignInRequired from "../comps/ui/SignInRequired";
+
+const useStyles = makeStyles(theme => ({
+	formContainer : {
+		maxWidth: "1200px",
+		margin: "auto",
+		padding: "2rem",
+	},
+	form: {
+		margin: "auto",
+		display: "block",
+		textAlign: "center",
+	}
+}));
+
+export default function FeedbackForm() {
+	const classes = useStyles();
+	if (!this.context.signedIn) {
```

Comment:
This won't work in a functional component. You need to import the context that you're trying to use (UserContext) in this case. Then use the React.useContext hook and pass in the context. You can see an example of this here: 
https://github.com/stuysu/stuyactivities.org/blob/master/src/pages/org/join.js#L53